### PR TITLE
Modify BUFR converter and filter yamls for MetoOp Scatwind

### PR DESCRIPTION
**Pull request by emilyhcliu**
Two updates for MetOp SCATWIND; one minor updates for GOES AMV 

1. BUFR Converter:
    - remove wind direction and speed
    - add `stationElevation` as zero 

2. UFO Filters:
    - add linear obs operator section
      The declaration of `linear obs operator" is necessary for GOES AMV since there is no 
      linearized component for applying the 10-meter factor.
    - add `Gaussian Thinning` filter to thin data with 75 km box (equal box size, prefer data close to central of the box)

3. For GOES AMV YAML files, change output data extension from `nc4` to `nc`

UFO vs. GSI O-F and QC comparison: O-F and data count passed look close
ufo_ascat_metop-b_omf_windEastward_qc (1)
gsi_ascat_metop-b_omf_windEastward_qc

3Dvar increments are reasonable with MetOp-B SCATWIND alone.
```
0: ----------------------------------------------------------------------------------------------------
  0: Increment print | number of fields = 8 | cube sphere face size: C768
  0: eastward_wind                                | Min:-1.741092e+00 Max:+1.712287e+00 RMS:+2.282584e-03
  0: northward_wind                               | Min:-1.825957e+00 Max:+1.930385e+00 RMS:+2.496075e-03
  0: air_temperature                              | Min:+0.000000e+00 Max:+0.000000e+00 RMS:+0.000000e+00
  0: surface_pressure                             | Min:+0.000000e+00 Max:+0.000000e+00 RMS:+0.000000e+00
  0: specific_humidity                            | Min:+0.000000e+00 Max:+0.000000e+00 RMS:+0.000000e+00
  0: cloud_liquid_ice                             | Min:+0.000000e+00 Max:+0.000000e+00 RMS:+0.000000e+00
  0: cloud_liquid_water                           | Min:+0.000000e+00 Max:+0.000000e+00 RMS:+0.000000e+00
  0: ozone_mass_mixing_ratio                      | Min:+0.000000e+00 Max:+0.000000e+00 RMS:+0.000000e+00
  0: ----------------------------------------------------------------------------------------------------

```

3DVar increments are also reasonable with MetOp-B + MetOp-A assimilated

### CODE REVIEW

File: ush/ioda/bufr2ioda/bufr2ioda_satwind_scat.py [hunk 1]
```diff
@@ -299,6 +303,12 @@ def bufr_to_ioda(config, logger):
                 .write_attr('long_name', 'Height of Observation') \
                 .write_data(height2)
 
+            # Station Elevation
```
Comment (CoryMartin-NOAA) [inline ush/ioda/bufr2ioda/bufr2ioda_satwind_scat.py]: I think the coding norms are complaining here?

Comment (CoryMartin-NOAA) [inline ush/ioda/bufr2ioda/bufr2ioda_satwind_scat.py]:
```suggestion
            # Station Elevation
```

**Review by CoryMartin-NOAA — APPROVED** [review]

**Review by RussTreadon-NOAA — APPROVED** [review]
Copied modified files to working copy of gdas-validation.  Ran gdasprepatmiodaobs.   satwind and scatwind pieces run to completion.   

Approve.

File: ush/ioda/bufr2ioda/bufr2ioda_satwind_scat.py (hunk 1 above)
Comment (emilyhcliu) [reply ush/ioda/bufr2ioda/bufr2ioda_satwind_scat.py]: I fixed the coding norm error.

**Pull request merged**